developers/integrations/index.md

```diff
@@ -6,25 +6,32 @@ image: og/integrations/home.jpg
 
 Weaviate's integration ecosystem enables developers to build various applications leveraging Weaviate and another technology.
 
-You can find all notebooks and examples on [Weaviate Recipes](https://github.com/weaviate/recipes)!
+All the notebooks and code examples are on [Weaviate Recipes](https://github.com/weaviate/recipes)!
+
+<img
+    src={require('./ecosystem.png').default}
+    alt="alt"
+    style={{ maxWidth: "60%", display: "block", marginLeft: "auto", marginRight: "auto"}}
+/>
+
 
 ## About the Categories
-The ecosystem is broken out by 5 categories:
+The ecosystem is broken into five categories:
 
-* **Cloud Hyperscalers** - 
-* **Cloud Platforms** - 
-* **Data Pipeline** - 
-* **LLM Frameworks** - 
-* **Observability and Evaluation** - 
+* **Cloud Hyperscalers** - Cloud service providers offering large-scale computing and storage
```

Comment:
The descriptions specify companies, tools, and platforms. Those categories might change over time (is langChain a data pipeline?) Maybe generalize to something like:

**Cloud Hyperscalers** - Large-scale computing and storage
**Container Infrastructure** - Run and scale containerized applications
**Data Pipeline** - Data ingestion and scraping 
**LLM Frameworks** - Build generative AI applications
**Observability and Evaluation** - Monitor and analyze generative AI workflows